website/README.md

```diff
@@ -0,0 +1,13 @@
+Drag and drop *image_drop.html* into your browser and you can interact with the 
```

Comment:
The README requires rewriting since the last changes were made.

It's currently difficult to make sense of where to start from.